Makefile

```diff
@@ -30,14 +30,16 @@ networks:
 	@docker network create --driver=bridge hashicorp || true
 	@docker network create --driver=bridge --subnet=172.16.237.0/24 besu || true
 	@docker network create --driver=bridge --subnet=172.16.238.0/24 quorum || true
+	@docker network create --driver=bridge qkm || true
```

Comment:
@ggarri If I remember correctly you told me to not do that for postgres and the qkm network

Reply:
Actually he is right, we need to init a `qkm` network to attach deps like postgres. What you did was the oppostive, you added a `postgres` network what didn't make sense imo. In addition to this I think we could remove `hashicorp` bridge an attach it also to `qkm` network, wdyt @dolanor ?

Reply:
From what I understand, we have 3 types of dependencies:
- mandatory upstream dependencies, like postgres: without it, QKM shouldn't start
- optional upstream dependencies, like hashicorp vault, AKM, AWS KMS: we need them only if the configuration requires it to.
- downstream dependencies: besu and quorum which are users of QKM.

For mandatory, we need the network, I would say it could even be part of the default docker-compose.yml since without it, QKM shouldn't start.
For optional, they could have their own independant/external network, and given the config, they will be requested or not.
For downstream, they clearly need their own network, they are users of QKM. QKM shouldn't even know about them.

So I'd suggest to put it directly in the default QKM docker-compose.yml and the network would start and die when we stop the QKM. (there is no point of having a solo postgres running without a qkm attached to it)

Reply:
And what I understand of the ownership of docker-compose.yml file
- [./docker-compose.yml](https://github.com/ConsenSys/quorum-key-manager/blob/0a286f9d66f357707eea64326114ff86d7ecc382/docker-compose.yml) is for QKM.
- [./deps/docker-compose.yml](https://github.com/ConsenSys/quorum-key-manager/blob/0a286f9d66f357707eea64326114ff86d7ecc382/deps/docker-compose.yml) is for QKM's mandatory dependencies.- [./deps/docker-compose.yml](https://github.com/ConsenSys/quorum-key-manager/blob/0a286f9d66f357707eea64326114ff86d7ecc382/deps/docker-compose-tools.yml) is for QKM mandatory dependencies' tools.
- [./deps/hashicorp/docker-compose.yml](https://github.com/ConsenSys/quorum-key-manager/blob/0a286f9d66f357707eea64326114ff86d7ecc382/deps/hashicorp/docker-compose.yml) is for QKM optional dependencies
- `./deps/{besu,quorum}/docker-compose.yml` is for QKM downstream dependencies (which are users of QKM)

Reply:
I'll put it in the root ./docker-compose.yml file after we'll discuss about it.